tools/svg2c/main.go

```
@@ -0,0 +1,298 @@
+		(maxPoint.X - minPoint.X) / 2,
+		(maxPoint.Y - minPoint.Y) / 2,
+	}
+	sd.Origin = Add(minPoint, midPoint)
+}
+
+func (sd *SvgData) CalcDataWithOffset() {
+	for idx, gData := range sd.Data {
+		sd.Data[idx].DeltaPoints = gData.GetWithOffsetAndDelta(sd.Origin)
+	}
+}
+
+func (sd *SvgData) GetMinPoint() Point {
+	var minPoint = sd.Data[0].Points[0]
+	for _, gData := range sd.Data {
+		for _, point := range gData.Points {
+			if point.X < minPoint.X {
+				minPoint.X = point.X
+			}
+			if point.Y < minPoint.Y {
+				minPoint.Y = point.Y
+			}
+		}
+	}
+	return minPoint
+}
+
+func (sd *SvgData) GetMaxPoint() Point {
+	var maxPoint = sd.Data[0].Points[0]
+	for _, gData := range sd.Data {
+		for _, point := range gData.Points {
+			if point.X > maxPoint.X {
+				maxPoint.X = point.X
+			}
+			if point.Y > maxPoint.Y {
+				maxPoint.Y = point.Y
+			}
+		}
+	}
+	return maxPoint
+}
+
+func (sd *SvgData) Export() (output string, err error) {
+	t, err := template.New("svg").Parse(svgTemplate)
+	if err != nil {
+		return
+	}
+
+	file, err := os.Create(outputPath)
+	if err != nil {
+		return "", err
+	}
+	defer file.Close()
+
+	var data bytes.Buffer
+	err = t.Execute(&data, sd)
+	if err != nil {
+		return "", err
+	}
+	return data.String(), nil
+}
+
+func (gd *GeometricData) GetWithOffsetAndDelta(offset Point) []Point {
+	inputPoints := gd.GetPointsWithOffset(offset)
+	deltaPoints := []Point{inputPoints[0]}
+	for index, point := range inputPoints {
+		if index != 0 {
+			lastPoint := inputPoints[index-1]
+			deltaPoints = append(deltaPoints, Sub(point, lastPoint))
+		}
+	}
+	return deltaPoints
+}
+
+func (gd *GeometricData) GetPointsWithOffset(offset Point) []Point {
+	var withOffset []Point
+	for _, point := range gd.Points {
+		withOffset = append(withOffset, Sub(point, offset))
+	}
+	return withOffset
+}
+
+func handleFile(file *os.File, name string) string {
+	svgData := SvgData{[]GeometricData{}, Point{0, 0}, name}
+
+	element, _ := svgparser.Parse(file, false)
```

Why are you discarding an error here?